site/routes/definitions.js

```diff
@@ -13,6 +13,7 @@ export const routeDefinitions : Array<RouteDefinition> = [
     title: 'Home',
     key: 'homePage',
     route: '',
+    stateToProps: state => ({ featuredBlogPosts: state.featuredBlogPosts }),
```

Comment:
`({ featuredBlogPosts }) => ({ featuredBlogPosts })` although maybe this is more readable?